packages/libpq/build

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# libpq is the postgresql client library. It is required to communicate with any
+# database that communicates using the postgresql dialect, for example CockroachDB.
+
+pushd "/pkg/src/libpq"
+
+find /opt/mesosphere/
```

Comment:
I realize this is a straight copy, but this displays the contents of the directory `/opt/mesosphere`. Is this required?

Reply:
@gpaul :-)